lib/players/computer.rb

```diff
@@ -0,0 +1,137 @@
+module Players
+  class Computer < Player
+
+    def computer_options(board)
+        move = ["7", "1", "9", "3", "4", "8", "6", "2", "5"]
+        moves = move.find {|slot| !board.taken?(slot)}
+        moves
+    end
+
+    def move(board)
```

Comment:
I would try to find a way to iterate over the WIN_COMBINATIONS for best possible win. This works but it is extremely verbose and a lot to read.